src/asciidoc/docs/about/contribute.adoc

```diff
@@ -0,0 +1,176 @@
+= Contributing to the Koji developer site
+:page-slug: contribute-koji-developers
+:page-banner: {imagesDir}/example.png
+
+The Koji for Developers documentation site is built and maintained by developers just like you.
+You too can help by contributing to the site *structure*, writing new *articles*, or *suggesting edits* to existing documentation.
+
+Anyone can open an issue about documentation, or contribute a change with a pull request to the `madewithkoji/docs` https://github.com/madewithkoji/docs[GitHub repository].
+
+link to code of conduct?
+
+include::../_includes/tip-discord-server.adoc[tags=*]
+
+== Editing the documentation
+
+The documentation is written in https://asciidoc.org/[Asciidoc] which is a lightweight markup language for simplifying the creation of documentation.
+All the AsciiDoc files that map to this site can be found in the repository under `src/asciidoc/`.
+
+Here's a quick example:
+
+[source, asciidoc]
+----
+= Koji for Developers
+Create your own templates for https://withkoji.com[Koji].
+
```

Comment:
Suggest adding a space after the h1 and removing the extra space below the preamble (to demonstrate Asciidoc standards in the example).